src/utils/AppStateMonitor.ts

```diff
@@ -57,13 +58,18 @@ export default class AppStateMonitor {
 
       // Trigger flag if the user had the app suspended for too long.
       if (
-        !experiment.isComplete &&
-        (suspendedTime > SUSPENDED_TIMEOUT ||
-          (applyStrictTimeout && suspendedTime > STRICT_SUSPENED_TIMEOUT))
+        experiment.breakEndDate === undefined ||
+        isPast(experiment.breakEndDate)
       ) {
-        store.dispatch(
-          rejectParticipant(applyStrictTimeout ? 'TRIAL_TIMEOUT' : 'TIMEOUT'),
-        )
+        if (
+          !experiment.isComplete &&
```

Comment:
Nitpick: These two if statements can be merged, but I don't mind them not being merged if you think that's more readable.

Reply:
I tidyied it up and extracted the conditionals to variables and then used a single `if` , 👍